Library/Formula/llvm.rb

```diff
@@ -198,12 +196,6 @@ def install
 
     args << "-DLLVM_ENABLE_RTTI=On" if build.with? "rtti"
 
-    if build.with? "assertions"
-      args << "-DLLVM_ENABLE_ASSERTIONS=On"
-    else
-      args << "-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'"
```

Comment:
Sorry if I wasn't clear. I was referring to the `-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'` argument, that would have been added previously when using the `--without-assertions` option. You're removing that one as well with this PR, not just the assertions-related stuff.